examples/htlc-pyteal/scripts/transfer/htlc-example.js

```diff
@@ -27,12 +27,12 @@ async function run(runtimeEnv, deployer) {
         toAccountAddr: john.addr,
         amountMicroAlgos: 200,
         lsig: lsig,
-        payFlags: {}
+        payFlags: {totalFee: 500}
```

Comment:
What happens if we don't specify the `totalFee`? Will the transaction fail?

Reply:
yes, transaction may fail and we may burn more algos than needed.
Teal Guidelines suggests to add a fee check in contracts to avoid burning extra algos.

Reply:
We have added a fee check in the smart contract (<= 10,000). 
I was asking what happens if i pass `payFlags: {}`. (i.e do we need to pass `totalFee: 500`) 
What's the default value of totalFee? I think it's `1000`. Can you confirm?

Reply:
yes, default value is set to 1000 by algob. But it may be higher if user wants. As per TEAL Guidelines we need to check this in contracts too. Is that what you are asking ?

Reply:
Yes. That's what i wanted to confirm. `1000` is the algorand's min default txn fee. Please leave any one txn like :
```
...
payFlags: {} // default totalFee of 1000 microalgos is used
...
```

Reply:
yes, looks good

Reply:
Also, should we add one txn with totalFee > 10,000 and failing the transaction. (indicating the user is burning too much algo) What do think?

Reply:
It seems there is a problem, i am getting rejected by logic when leaving payFlags empty. That means somehow fee is being set to greater than 10000.
could you please validate ?

Reply:
![sample](https://user-images.githubusercontent.com/26249903/100992134-d1aca680-3579-11eb-94f9-2bd31fbdcdff.png)
Look at this transaction, fee is very high, because it was left blank

Reply:
Ok let me check as well

Reply:
I don't think minimum default fee is 1000, we are getting fee value from `const s = await getSuggestedParams(algocl);`

Reply:
With me also it's getting set to `267000`.